runtime/vm/resolvesupport.cpp

```diff
@@ -1611,7 +1700,9 @@ resolveVirtualMethodRefInto(J9VMThread *vmStruct, J9ConstantPool *ramCP, UDATA c
 				}
 			}
 		}
-#endif /* J9VM_OPT_METHOD_HANDLE */
+#else /* defined(J9VM_OPT_METHOD_HANDLE) */
+	Trc_VM_Assert_ShouldNeverHappen();
```

Comment:
Does this need to be indented 1 more level to match the surrounding code?
```suggestion
		Trc_VM_Assert_ShouldNeverHappen();
```